GNUmakefile

```diff
@@ -13,5 +13,8 @@ help:
 	@echo "Please use 'make <target>' where <target> is one of.."
 	@echo "  test    Runs all tests in tests/ directory."
 
+egg:
```

Comment:
ok, there actually needs to be more to egg building.  i think in setup.py we'd need to make some changes.  just remove this part for now.  keep the test section though